src/lib/settingsmanager.ts

```diff
@@ -26,6 +26,20 @@ export function getControllerConfigDropdownValues(): Array<{ id: keyof typeof Co
   });
 }
 
+export enum VideoCodecValues {
+  Default = "Default",
+  AV1 = "AV1",
+  HEVC = "HEVC",
+  H264 = "H.264",
```

Comment:
```suggestion
  "H.264" = "H264",
```
The enum in typescript is just an object so we can use any string we want as a key (which can then be accessed as `VideoCodecValues["H.264"]`.

Reply:
I actually looked for a way of doing this but didn't find it. Thanks for the suggestion!